subworkflows/local/kraken2_classification/main.nf

```diff
@@ -0,0 +1,103 @@
+include { KRAKEN2_KRAKEN2 } from '../../../modules/nf-core/kraken2/kraken2/main'
+include { KRAKEN_PARSE    } from '../../../modules/local/kraken/parse/main'
+include { KRAKEN_MERGE    } from '../../../modules/local/kraken/merge/main'
+include { SOURCEPREDICT   } from '../../../modules/nf-core/sourcepredict/main'
+include { UNTAR           } from '../../../modules/nf-core/untar/main'
+include { XZ_DECOMPRESS   } from '../../../modules/nf-core/xz/decompress/main'
+
+if (params.sp_sources  )              { ch_sp_sources = file(params.sp_sources) } else { error("SourcePredict sources file not specified!") }
+if (params.sp_labels   )              { ch_sp_labels  = file(params.sp_labels) } else { error("SourcePredict labels file not specified!") }
+if (params.taxa_sqlite_traverse_pkl ) { ch_sqlite_traverse = file(params.taxa_sqlite_traverse_pkl) } else { error("Ete3 taxa.sqlite.traverse file not specified!") }
```

Comment:
Can you add the following checks to these parameters in `nextflow_schema.json`?
```
"format": "file-path"
"exists": true
```

You can remove these checks completely then as nf-schema will fully cover this